scripts/hurricane/stats/exevs_hurricane_regional_tropcyc_stats.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 export PS4=' + exevs_hurricane_regional_tropcyc_stats.sh line $LINENO: '
 
-export MetOnMachine=$MET_ROOT
+export MetOnMachine=/apps/ops/prod/libs/intel/19.1.3.304/met/10.1.2
 export LEAD_List="-lead 000000 -lead 060000 -lead 120000 -lead 180000 -lead 240000 -lead 300000 -lead 360000 -lead 420000 -lead 480000 -lead 540000 -lead 600000 -lead 660000 -lead 720000 -lead 780000 -lead 840000 -lead 900000 -lead 960000 -lead 1020000 -lead 1080000 -lead 1140000 -lead 1200000 -lead 1260000"
```

Comment:
You will want to get this change back into your PR. You accidentally introduced older code for line 4.